### PR TITLE
Computation of Omega groups of arbitrary p groups is supported.

### PR DESCRIPTION
**Pull request by DominikBernhardt**
The manual claims GAP can only compute Omega-groups of abelian p groups. In my opinion, this is not true, see `lib/grppc.gi`. This PR removes the corresponding line in the manual.
Also, a part of the comment in the code was redundant: Any p-group with nilpotency class at most `p` is regular, so I clarified this part in the documentation.

I put Release notes: To be added. Feel free to change this if you want to.

### CODE REVIEW

File: lib/grp.gd [hunk 1]
```diff
@@ -1972,7 +1972,6 @@ DeclareAttribute( "NrConjugacyClasses", IsGroup );
 ##  \{ g \in <A>G</A> \mid g^{{<A>p</A>^{<A>n</A>}}} = 1 \}</M>.
 ##  The default value for <A>n</A> is <C>1</C>.
 ##  <P/>
-##  <E>@At the moment methods exist only for abelian <A>G</A> and <A>n</A>=1.@</E>
```
Comment (fingolfin) [inline lib/grp.gd]: OK, I tried this and it seemed to work:
```
gap> grps:=AllSmallGroups(32);;
gap> Collected(List(grps,g->Size(Omega(g,2))));
[ [ 2, 2 ], [ 4, 14 ], [ 8, 10 ], [ 16, 13 ], [ 32, 12 ] ]
gap> Collected(List(grps,g->Size(Omega(g,2,2))));
[ [ 4, 1 ], [ 8, 3 ], [ 16, 9 ], [ 32, 38 ] ]
gap> Collected(List(grps,g->Size(Omega(g,2,3))));
[ [ 8, 1 ], [ 16, 2 ], [ 32, 48 ] ]
```
So it seems you are right. But perhaps add a minimal test to verify this? Anyway, that's optional.

**Comment by fingolfin** [comment]
I don't think this needs to go into the release notes; it just removes an incorrect restriction in the manual, which did log a bit odd to start with. The code itself has been in there (apparently written by Jack Schmidt and added in by @hulpke )

**Comment by DominikBernhardt** [comment]
@fingolfin Added one test.

File: tst/testinstall/grppc.tst [hunk 2]
```diff
@@ -166,6 +166,12 @@ gap> RepresentativeAction(G,x,x^2)<>fail;
 true
 gap> RepresentativeAction(G,x,x^2);
 f1*f2
+gap> g := SmallGroup(243,27);
+<pc group of size 243 with 5 generators>
+gap> Omega(g,3,1);
```
Comment (fingolfin) [inline tst/testinstall/grppc.tst]: The way the output is printed could change, and it's not even clear to me that it is correct now. How about this instead (and similar below):
```suggestion
gap> AsSet(Omega(g,3,1)) = Set(Filtered(g, g -> IsOne(g^3)));
true
```

Comment (DominikBernhardt) [reply tst/testinstall/grppc.tst]: I was just stupid there, sorry. I changed the test.

**Pull request merged**